.github/workflows/php-tests.yml

```diff
@@ -0,0 +1,70 @@
+name: PHP Tests
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - ready_for_review
+
+jobs:
+  php-tests:
+    name: Run PHP Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: ['5.6', '7.4']
+        wordpress: ['4.7', 'latest-stable', 'latest-stable (multisite)', 'nightly']
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_DATABASE: wordpress
+    steps:
+      - uses: actions/checkout@v2
+      - uses: shivammathur/setup-php@v2
+        with:
+          extensions: mbstring, intl, redis, pdo_mysql
+          php-version: ${{ matrix.php }}
+      - name: Start MySQL
+        run: sudo service mysql start
+      - name: Enable multisite
+        if: ${{ matrix.wordpress == 'latest-stable-multisite' }}
+        run: |
+          echo "WP_MULTISITE=1" >> $GITHUB_ENV
+      - name: Extract version from stable WordPress
+        if: ${{ matrix.wordpress == 'latest-stable' || matrix.wordpress == 'latest-stable-multisite' }}
+        run: |
+          curl -s http://api.wordpress.org/core/version-check/1.7/ > /tmp/wp-latest.json
+          echo "WP_VERSION=$(grep -o '"version":"[^"]*' /tmp/wp-latest.json | sed 's/"version":"//')" >> $GITHUB_ENV
+        env:
+          WP_VERSION: ${{ matrix.wordpress }}
+      - name: Set WordPress version
+        if: ${{ !(matrix.wordpress == 'latest-stable' || matrix.wordpress == 'latest-stable (multisite)') }}
+        run: echo "WP_VERSION=$WP_VERSION" >> $GITHUB_ENV
+        env:
+          WP_VERSION: ${{ matrix.wordpress }}
```

Comment:
I don't think we need it via env either - it's only needed for passing to the `tests/bin/install-wp-tests.sh` script.